frontend/packages/ux-editor/src/components/config/editModal/EditComponentId.tsx

```diff
@@ -16,45 +14,39 @@ export const EditComponentId = ({
   handleComponentUpdate,
   helpText,
 }: IEditComponentId) => {
-  const { components, containers } = useSelectedFormLayout();
-  const { t } = useTranslation();
-
   const handleIdChange = (id: string) => {
     handleComponentUpdate({
       ...component,
       id,
     });
   };
 
+  // TODO
+  // Missing:
+  // - helpText
+  // - Validation/Error handling
+  // - onBlur improvement
```

Comment:
Added a comment with a suggestion on how to improve the `onBlur` behaviour in the `StudioTextField` component 😊